ui/src/features/get-jobs.js

```diff
@@ -24,11 +24,12 @@ import { ReactTableComponent } from '../components/ReactTable';
 import { getColumns } from './configurationColumn';
 import _ from 'lodash';
 import {createJobRequest} from './requestBuilder';
```

Comment:
For a perfect PR- missing a feedback message like "edited successfully " and error handling and message for a case of an error.

I appreciate your work! , thanks!

Reply:
Thank you for your kind words, i added success message as well as error handling.